client/my-sites/site-settings/redirect-to/README.md

```diff
@@ -0,0 +1,47 @@
+# Redirect To
+
+This component provides automatic (prior to rendering the site) redirection for non-Jetpack sites.
+Here, we consider Atomic sites to be Jetpack-like.
+
+## Props passed to the component
+### `WrappedComponent`
+
+Component to wrap.
+
+### `redirectUrl`
```

Comment:
Naming nit: it's not a URL, it's more of a route, right? One may pass a URL and put the app in an unexpected state if they use a URL here.